docs/FeatureManagement/FeatureFlagv2.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v2.0.0/FeatureFlag.json",
+    "type": "object",
+    "title": "An Azure App Configuration Feature Declaration",
+    "required": [
+      "id",
+      "enabled",
```

Comment:
If the `enabled` property has a default value, it is not required. Do we require it in our provider/FM libraries?

Reply:
Portal doesn't let you omit enabled, but the provider will just set it to false by default

Reply:
If the library doesn't enforce it, it's not required. Looks like our [new schema support PR](https://github.com/microsoft/FeatureManagement-Dotnet/pull/319/files#diff-55378af224e45c1157f406ef1f2714ddf93ee9ac4f312f5d6a4e6cf0fd58e72bR339) doesn't require it either. 

We should remove it from the required and all FM libraries should treat a feature flag as disabled if the property is not present.